### PR TITLE
Live from Processing Day!

### PR DESCRIPTION
**Pull request by shiffman**
Stil live from Processing day. This is a cloud that I made as an example.

### CODE REVIEW

File: generators.js [hunk 1]
```diff
@@ -6,6 +6,28 @@
 // register(perlinNoiseCloud, "Perlin Noise Cloud", "Daniel Shiffman");
 // Here's an example below!
 
+function PCDCloud() {
+  translate(width / 2, height / 2);
+  strokeWeight(1);
+  noFill();
+  let stepsize = 5;
```
Comment (shiffman) [inline generators.js]: maybe you should learn about const?

**Comment by meiamsome** [comment]
Looks like you need to set angle mode to degrees!

**Review by meiamsome — CHANGES_REQUESTED** [review]
Needs an explicit `angleMode(DEGREES)` at the top of the function!

**Comment by shiffman** [comment]
Ok, I added `angleMode(DEGREES)` and also moved the function below the example, thanks for reviewing!

**Pull request merged**